pyproject.toml

```diff
@@ -1,2 +1,2 @@
 [tool.black]
-target-version = ["py37", "py38"]
+target-version = ["py38"]
```

Comment:
```suggestion
target-version = ["py38", "py39"]
```

Reply:
I tried this but py39 didn't work: https://github.com/invenia/TagBotGitLab/runs/1704491625?check_suite_focus=true

Reply:
ohh yeah sorry. I tried doing this in a personal project once and it didn't work also. I think black hasn't added that yet. Just keep it at `py38` only then.